src/hubbleds/pages/03-distance-measurements/__init__.py

```diff
@@ -0,0 +1,342 @@
+            )
+            ScaffoldAlert(
+                GUIDELINE_ROOT / "GuidelineAngsizeMeas2b.vue",
+                event_next_callback=lambda *args: component_state.transition_next(),
+                event_back_callback=lambda *args: component_state.transition_previous(),
+                can_advance=component_state.can_transition(next=True),
+                show=component_state.is_current_step(Marker.ang_siz2b),
+            )
+            ScaffoldAlert(
+                #FIX This will need to be wired up once measuring tool is implemented
+                GUIDELINE_ROOT / "GuidelineAngsizeMeas3.vue",
+                event_next_callback=lambda *args: component_state.transition_next(),
+                event_back_callback=lambda *args: component_state.transition_previous(),
+                can_advance=component_state.can_transition(next=True),
+                show=component_state.is_current_step(Marker.ang_siz3),
+            )
+            ScaffoldAlert(
+                #FIX This will need to be wired up once measuring tool is implemented
+                GUIDELINE_ROOT / "GuidelineAngsizeMeas4.vue",
+                event_next_callback=lambda *args: component_state.transition_next(),
+                event_back_callback=lambda *args: component_state.transition_previous(),
+                can_advance=component_state.can_transition(next=True),
+                show=component_state.is_current_step(Marker.ang_siz4),
+            )
+            ScaffoldAlert(
+                GUIDELINE_ROOT / "GuidelineAngsizeMeas5a.vue",
+                event_next_callback=lambda *args: component_state.transition_next(),
+                event_back_callback=lambda *args: component_state.transition_previous(),
+                can_advance=component_state.can_transition(next=True),
+                show=component_state.is_current_step(Marker.ang_siz5a),
+                state_view={
+                    "dosdonts_tutorial_opened": component_state.dosdonts_tutorial_opened.value
+                },
+            )
+            # This was skipped in voila version
+            # ScaffoldAlert(
+            #     GUIDELINE_ROOT / "GuidelineAngsizeMeas6.vue",
+            #     event_next_callback=lambda *args: component_state.transition_next(),
+            #     event_back_callback=lambda *args: component_state.transition_previous(),
+            #     can_advance=component_state.can_transition(next=True),
+            #     show=component_state.is_current_step(Marker.ang_siz6),
+            # )
+            ScaffoldAlert(
+                GUIDELINE_ROOT / "GuidelineDotplotSeq5.vue",
+                event_next_callback=lambda *args: component_state.transition_next(),
+                event_back_callback=lambda *args: component_state.transition_previous(),
+                event_jump_rep_rem1=lambda *args: component_state.transition_to(Marker.rep_rem1),
+                can_advance=component_state.can_transition(next=True),
```

Comment:
```suggestion
                can_advance=False,
```

Reply:
This can just be false, that way the user only has the buttons to press. `can_advance` acts kind of like hiding the next button